inc/scss-compiler.php

```diff
@@ -62,7 +63,11 @@ function bootscore_compile_scss() {
       set_theme_mod('bootscore_scss_modified_timestamp', $last_modified);
     }
   } catch (Exception $e) {
-    wp_die('<b>bootScore SCSS Compiler - Caught exception:</b><br><br> ' . $e->getMessage());
+    if ($is_environment_dev) {
+      wp_die('<b>bootScore SCSS Compiler - Caught exception:</b><br><br> ' . $e->getMessage());
+    } else { // supress error message for production
+      wp_die('<b>SCSS ERROR!</b>');
```

Comment:
A nicer error message would be better.
```suggestion
      wp_die('Something went wrong with the SCSS compiler.');
```